File: src/main/java/com/hiddewieringa/elevator/domain/elevator/query/QueryFloor.java
package com.hiddewieringa.elevator.domain.elevator.query;

import com.hiddewieringa.elevator.domain.elevator.model.ElevatorId;
import org.axonframework.commandhandling.TargetAggregateIdentifier;

public class QueryFloor {

    @TargetAggregateIdentifier
    private final ElevatorId elevatorId;

    public QueryFloor(ElevatorId elevatorId) {
        this.elevatorId = elevatorId;
    }

    public ElevatorId getElevatorId() {
        return elevatorId;
    }

}

File: src/main/kotlin/com/hiddewieringa/elevator/domain/elevator/query/QueryFloor.kt
package com.hiddewieringa.elevator.domain.elevator.query

import com.hiddewieringa.elevator.domain.elevator.model.ElevatorId
import org.axonframework.modelling.command.TargetAggregateIdentifier

class QueryFloor(
    @field:TargetAggregateIdentifier
    val elevatorId: ElevatorId
)
